src/base/quda_bridge.cpp

```diff
@@ -429,9 +429,10 @@ namespace quda_iface
     inv_param.output_location=QUDA_CPU_FIELD_LOCATION;
     
     //inv_param.tune=QUDA_TUNE_YES;
-    
-    inv_param.sp_pad=0;
```

Comment:
@Marcogarofalo ah, that's a good catch, I forgot that this happened at some point in QUDA. Didn't realise that Nissa had not adapted to this.

Reply:
that was caught by the compiler.